ls1intum/Artemis
# `Modeling exercises`: Bump Apollon editor to version 3.0.0-beta.1

This PR updates the version of Apollon used in Artemis to 3.0.0-beta.1. This Apollon update adapts the format how diagrams are stored, adds support BPMN diagrams in a beta version and allows user to zoom in and out on the canvas.

### Checklist
#### General



### Motivation and Context
This PR is intended to update Artemis with the latest version of Apollon.

### Steps for Testing
When testing this change, the following 4 areas are critical. 

1. Modeling by student in course and exam exercises
2. Creation of example solution by instructors
3. Assessment of a model by tutors
4. Creation of a quiz drag and drop exercises by instructors

For all these, it should be ensured if Artemis still behaves as expected in regards to the Apollon integration and if the zoom-in and zoom-out functionalities are available and operating as expected on the Apollon canvas. Additionally, BPMN diagrams should be available for creation but marked as still in beta.
